### PR TITLE
Use locale info to create default config

### PR DESCRIPTION
**Pull request by ThomasdenH**
Issue #736, get measurement system, temperature system, language and
currency from locale and use it as default.

### CODE REVIEW

**Comment by ThomasdenH** [comment]
It is building now, but since there are some warnings, I'll try to do some cleaning up.

File: src/platform/windows.c [hunk 1]
```diff
@@ -591,4 +594,125 @@ PCHAR *CommandLineToArgvA(PCHAR CmdLine, int *_argc)
...
+		sizeof(country)) == 0){
+		return TEMPERATURE_FORMAT_C;
+	}
+	switch (country){
+	case CTRY_UNITED_STATES:
+		//USA
+		return TEMPERATURE_FORMAT_F;
+	case CTRY_BELIZE:
+		// Belize
+		return TEMPERATURE_FORMAT_F;
```
Comment (IntelOrca) [inline src/platform/windows.c]: I wouldn't bother sticking a comment for what is directly written above.
Also you can put a case directly under another:

```c
case CTRY_UNITED_STATES:
case CTRY_BELIZE:
        return TEMPERATURE_FORMAT_F;
default:
        return TEMPERATURE_FORMAT_C;

**Comment by ThomasdenH** [comment]
You're right. I'll change that.

**Comment by ThomasdenH** [comment]
It can be merged now. Sorry for the many commits, but testing this was harder than I imagined.

**Pull request merged**